accelerator/common_core.c

```diff
@@ -450,3 +453,40 @@ status_t ta_send_bundle(const iota_config_t* const tangle, const iota_client_ser
 
   return SC_OK;
 }
+
+status_t ta_get_bundle_by_addr(const iota_client_service_t* const service, flex_trit_t const* const addr,
+                               bundle_transactions_t* bundle) {
+  status_t ret = SC_OK;
+  tryte_t bundle_hash[NUM_TRYTES_BUNDLE];
+  find_transactions_req_t* txn_req = find_transactions_req_new();
+  find_transactions_res_t* txn_res = find_transactions_res_new();
+  ta_find_transaction_objects_req_t* obj_req = ta_find_transaction_objects_req_new();
```

Comment:
Check `txn_req`, `txn_res` and `obj_req`

Reply:
I don't get it

Reply:
```suggestion
  if (obj_req ==NULL || ...) {
  }
```